spec/models/form/sales/pages/person_known_spec.rb

```diff
@@ -86,15 +65,27 @@
       end
 
       it "has correct depends_on" do
-        expect(page.depends_on).to eq(
-          [{ "hholdcount" => 3, "jointpur" => 2 }, { "hholdcount" => 4, "jointpur" => 2 }],
-        )
+        expect(page.depends_on).to eq([
+          {
+            "hholdcount" => {
```

Comment:
Again not a huge deal but the order of the items in these hashes are reversed compared with in `person_known.rb`. In there `jointpur` comes before `hholdcount` and `operator` comes before `operand`. Whereas here `jointpur` comes _after_ `hholdcount` and `operator` comes _after_ `operand`. Probs not worth changing unless you think so.